evergreen.yaml

```diff
@@ -71,6 +79,11 @@ functions:
       params:
         working_dir: certdepot
         command: make check-mongod
+    - command: subprocess.exec
+      type: setup
+      params:
+        command: make init-rs
+        working_dir: certdepot
```

Comment:
Is it necessary for the db to be a replica set? Or is it just for consistency?

Reply:
The tests don't work if I don't call init-rs on the latest version of Mongo? I think that might be the default configuration but I'm not sure. In any event it's consistent with what Evergreen does.

Reply:
Ah this is probably because I've been copying the mongo start code and it's subtly different between repos... exciting.